mavros/src/plugins/setpoint_attitude.cpp

```diff
@@ -18,6 +18,7 @@
 #include <mavros/mavros_plugin.h>
 #include <mavros/setpoint_mixin.h>
 #include <eigen_conversions/eigen_msg.h>
+#include <tf/transform_datatypes.h>
```

Comment:
you need to add `mav_msgs` as a dependency on `CMakeLists.txt` and `package.xml`